doc/syscall-limitations.md

```diff
@@ -52,20 +52,13 @@ on lowering the incompatibilities to enable more applications.
 | SYS_sched_setscheduler / SYS_sched_getscheduler | get/set the scheduling policy and params of process | Stub only |
 | SYS_sched_setattr / SYS_sched_getattr  | get/set scheduling policy and attributes of process | Stub only |
 | SYS_sched_get_priority_max / SYS_sched_get_priority_min  | get min/max priority levels of a policy | Stub only |
-| SYS_sched_setaffinity / SYS_sched_getaffinity | get/set a process's CPU affinity mask | Stub only |
 
 ## System calls related to user/group identity or identity-based permissions
 
 
 | Syscall names        | Description             | Compatibility |
 | -------------------- |-------------------| --------------|
-| SYS_chown / SYS_fchown / SYS_lchown | change owner identity of files | Unsupported |
-| SYS_umask | change default permissions of new files |  Unsupported |
-| SYS_getpgrp / SYS_setpgrp / SYS_getpgid / SYS_setpgid / SYS_getregid / SYS_setregid / SYS_getresgid / SYS_setresgid | get/set process's group ID | Unsupported |
-| SYS_getgroups / SYS_setgroups | get/set group IDs the process belongs to | Unsupported |
-| SYS_setsid / SYS_getsid | Create-and-set or get a session ID | Unsupported |
-| SYS_setreuid / SYS_setresuid/ SYS_getresuid | get/set process's user ID | Unsupported |
-| SYS_setfsuid  / SYS_setfsgid | set process's File System user/group identity | Unsupported |
+| SYS_setsid | Create-and-set a session ID | Unsupported |
```

Comment:
uid/gid/group should be added back... they set the identity on the thread, but ramfs and ext2 do not enforce identitity, hostfs only some APIs enforce. Most other syscalls that should do identity checks against these are not enforced.